assets/landscapes/Weather Report_files/index.js

```diff
@@ -0,0 +1,87 @@
+let initTemp = 71;
+const city = 'Seattle';
+
+document.getElementById('currentTemp').innerHTML = initTemp;
+
+cityValue = document.getElementById('inputCity').value = city;
+document.getElementById('city1').innerHTML = `${cityValue}✨`;
+
+// const state = {
+//   currentTemp: (document.getElementById('currentTemp').innerHTML = initTemp),
+// };
+
+// document.getElementById('currentTemp';
+
+// const changeTemp = () => {
+//   console.log('hello');
+//   var tempToChange = document.getElementById('currentTemp').innerHTML;
+//   console.log(tempToChange);
+//   let increment = tempToChange;
+//   console.log(increment);
+//   ++increment;
+//   console.log(increment);
+//   document.getElementById('currentTemp').innerHTML = increment;
+// };
+
+//printing default value of data that is 0 in h2 tag
+document.getElementById('currentTemp').innerHTML = initTemp;
+
+//creation of increment function
+function increment() {
+  initTemp += 1;
+  document.getElementById('currentTemp').innerHTML = initTemp;
+}
+//creation of decrement function
+function decrement() {
+  initTemp -= 1;
+  document.getElementById('currentTemp').innerHTML = initTemp;
+}
```

Comment:
Just a note that these can be combined into a single function that takes a parameter.